src/server/qgsserverrequest.h

```diff
@@ -90,6 +90,8 @@ class SERVER_EXPORT QgsServerRequest
       */
     QgsServerRequest::Method method() const;
 
+    bool fake() { return true; }
```

Comment:
What's that for?

Reply:
It was just to test Travis configuration, I'll remove it :)